package/ci/travis-emscripten.sh

```diff
@@ -19,16 +26,16 @@ cd ..
 # Crosscompile
 mkdir build-emscripten && cd build-emscripten
 cmake .. \
+    -DCMAKE_MAKE_PROGRAM=ninja \
     -DCORRADE_RC_EXECUTABLE=$HOME/deps-native/bin/corrade-rc \
     -DCMAKE_TOOLCHAIN_FILE="../toolchains/generic/Emscripten-wasm.cmake" \
-    -DEMSCRIPTEN_PREFIX=$(echo /usr/local/Cellar/emscripten/*/libexec) \
     -DCMAKE_BUILD_TYPE=Release \
     -DCMAKE_CXX_FLAGS_RELEASE="-DNDEBUG -O1" \
     -DCMAKE_INSTALL_PREFIX=$HOME/deps \
     -DCMAKE_EXE_LINKER_FLAGS_RELEASE="-O1" \
     -DBUILD_TESTS=ON \
     -G Ninja
-ninja -j4
+ninja -j2
```

Comment:
Just  `ninja`, it should figure out the cores on its own. 

I faintly remember I had to do this because otherwise at some point it was running 8 jobs at once, running out of memory. Probably not an issue nowadays anymore.